omemo/encryption.py

```diff
@@ -92,4 +92,20 @@ def migrate(dbConn):
                                      END TRANSACTION;
                                  """ % (create_table))
 
+    if user_version(dbConn) < 2:
+        delete_dupes = """ DELETE FROM identities WHERE _id not in (
+                            SELECT MIN(_id)
+                            FROM identities
+                            GROUP BY
+                            recipient_id, public_key
+                            );
+                        """
+
```

Comment:
> i revised the function who writes into that table to check for jid AND key, before writing.
the function is saveIdentity()

This is good, but still add an `UNIQUE` index to the sqlite migration. This make the code less errorprone, if i.e: `saveIdentity()` is changed. Also it follows the logic that the db is responsible for storing and maintaining data.